Development/nmos/streamcompatibility_api.h

```diff
@@ -0,0 +1,46 @@
+#ifndef NMOS_FLOWCOMPATIBILITY_API_H
+#define NMOS_FLOWCOMPATIBILITY_API_H
+
+#include <boost/variant.hpp>
+#include "bst/optional.h"
+#include "nmos/api_utils.h"
+#include "nmos/slog.h"
+
+// Stream Compatibility Management API implementation
+// See https://github.com/AMWA-TV/is-11/blob/v1.0-dev/APIs/FlowCompatibilityManagementAPI.raml
```

Comment:
A little while ago, all spec URLs were changed in nmos-cpp to use the published ones on specs.amwa.tv.

```suggestion
// See https://specs.amwa.tv/is-11/branches/v1.0-dev/APIs/StreamCompatibilityManagementAPI.html
```